src/kirby/components/card/card.component.tns.html

```diff
@@ -1,5 +1,5 @@
 <FlexboxLayout (loaded)="onViewLoaded($event)" class="background" [ngClass]="cardSizeClass" [kirbyElevation]="elevation">
-    <FlexboxLayout class="container" [ngClass]="themeColor">
+    <FlexboxLayout class="container {{themeColor}}" [ngClass]="{'padding': hasPadding}">
```

Comment:
We have found that the use of `[ngClass]="{...}"` binding can have performance impacts on native. So i suggest that you use the `[class.padding]` notation instead.

Reply:
Ok, will change that all places.